lib/quintype/api.rb

```diff
@@ -199,6 +199,8 @@ def _post_returning_headers(url_path, body, session_cookie=nil)
 
     def _get(url_path, *args)
       response = @@conn.get(@@api_base + url_path, *args)
+      return nil if response.status >= 400
```

Comment:
I checked against 200 at first, but wasn't sure if the api wouldn't return a 302. I think there is some api somewhere in sketches that does return a  302.